frontend/src/app/authentication/authorisation-interceptor.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpEvent, HttpHandler, HttpInterceptor, HttpRequest } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { LoginComponent } from './login/login.component';
+import { Observable } from 'rxjs';
+
+@Injectable(
+{
+	providedIn:'root'
+})
+export class AuthorisationInterceptor implements HttpInterceptor
+{
+	constructor(private loginComponent : LoginComponent)
+	{
+	}
+
+	intercept(request : HttpRequest<any>, next : HttpHandler) : Observable<HttpEvent<any>>
```

Comment:
you know that any is not allowed... CI should fail here

Reply:
I know but I don't know what to set there, I get like this from the documentation